java.jdbc-ballerina/Package.md

```diff
@@ -1,96 +1,17 @@
-[jdbc:Options](https://ballerina.io/learn/api-docs/ballerina/#/java.jdbc/records/Options)
-and those properties will be used by the defined `datasourceName`. 
-As per the provided example, the `org.h2.jdbcx.JdbcDataSource` datasource  will be configured with a `loginTimeout` 
-of `2000` milli seconds.
-
-```ballerina
-jdbc:Client|sql:Error dbClient1 = new ("jdbc:h2:~/path/to/database");
-jdbc:Client|sql:Error dbClient2 = new ("jdbc:h2:~/path/to/database", 
-                            "root", "root");
-jdbc:Client|sql:Error dbClient3 = new (url =  "jdbc:h2:~/path/to/database",
-                             user = "root", password = "root",
-                             options = {
-                                 datasourceName: "org.h2.jdbcx.JdbcDataSource"
-                             },
-                             connectionPool = {
-                                 maxOpenConnections: 5
-                             });
-jdbc:Client|sql:Error dbClient4 = new (url =  "jdbc:h2:~/path/to/database", 
-                             user = "root", password = "root",
-                             options = {
-                                datasourceName: "org.h2.jdbcx.JdbcDataSource", 
-                                properties: {"loginTimeout": "2000"}
-                             });                          
-```
-
-You can find more details about each property in the
-[jdbc:Client](https://ballerina.io/learn/api-docs/ballerina/#/java.jdbc/clients/Client) constructor. 
-
-The [jdbc:Client](https://ballerina.io/learn/api-docs/ballerina/#/java.jdbc/clients/Client) references 
-[sql:Client](https://ballerina.io/learn/api-docs/ballerina/#/sql/abstractObjects/Client) and 
-all the operations defined by the `sql:Client` will be supported by the `jdbc:Client` as well. 
-
-For more information on all the operations supported by the `jdbc:Client`, which include the below, see the
-[SQL Package]((https://ballerina.io/learn/api-docs/ballerina/#/sql)).
-
-1. Connection Pooling
-2. Querying data
-3. Inserting data
-4. Updating data
-5. Deleting data
-6. Batch insert and update data
-7. Execute stored procedures
-8. Closing client
-
-For examples on the usage of the connector, see the following.
-* [Client Initialization Example](https://ballerina.io/learn/by-example/jdbc-init-options.html) 
-* [Select Query Operations Example](https://ballerina.io/learn/by-example/jdbc-query-operation.html)
-* [Query with Complex Types Example](https://ballerina.io/learn/by-example/jdbc-complex-type-queries.html)
-* [DDL and DML Operations Example](https://ballerina.io/learn/by-example/jdbc-execute-operation.html)
-* [Batch Execute Operations Example](https://ballerina.io/learn/by-example/jdbc-batch-execute-operation.html)
-* [Parameterized Query Example](https://ballerina.io/learn/by-example/jdbc-parameterized-query.html)
+### Useful Links
+- Discuss the code changes of the Ballerina project in ballerina-dev@googlegroups.com.
```

Comment:
```suggestion
- Discuss code changes of the Ballerina project in ballerina-dev@googlegroups.com.
```

Reply:
resolved in https://github.com/ballerina-platform/module-ballerinax-java.jdbc/pull/160/commits/6eb3829883404ac0121c3854edf8f3a8988570d7